pkg/setting/setting.go

```diff
@@ -1933,6 +1939,19 @@ func (cfg *Cfg) readDataSourcesSettings() {
 	cfg.DataSourceLimit = datasources.Key("datasource_limit").MustInt(5000)
 }
 
+func (cfg *Cfg) readDataSourceSecuritySettings() {
+	datasources := cfg.Raw.Section("datasources.ip_range_security")
+	cfg.IPRangeACEnabled = datasources.Key("enabled").MustBool(false)
+	cfg.IPRangeACSecretKey = datasources.Key("secret_key").MustString("")
+	allowedURLString := datasources.Key("allow_list").MustString("")
+	for _, url := range strings.Split(allowedURLString, ",") {
```

Comment:
Nit. I think we have a string split function in util package that you might want to use. Think we use that for parsing other settings that's delimited.